docs/documentation-styleguide.md

```diff
@@ -0,0 +1,185 @@
+# Carbon Documentation Styleguide
+## Follow these guidelines when contributing documentation to the Carbon repository
+
+### Contents
+
+[Introduction](#introduction)
+
+[What We Document](#what-we-document)
+* [User Facing Documentation](#user-facing-documentation)
+* [Release Notes and Changelog](#release-notes-and-changelog)
+* [Request For Change (RFC)](#request-for-change)
+* [README files](#readme-files)
+* [Setup and Getting Started Guides](#setup-and-getting-started-guides)
+* [Styleguides](#styleguides)
+* [Other Internal Documentation](#other-internal-documentation)
+  * [New Joiners Guide](#new-joiners-guide)
+  * [Carbon Roadmap](#carbon-roadmap)
+  
+[Language and Grammar Style](#language-and-grammar-style)
+* [Jargon](#jargon)
+* [Abbreviations (including Acronyms)](#abbreviations-including-acronyms)
+* [Capitalisation](#capitalisation)
+
+[Text Formatting and Organisation](#text-formatting-and-organisation)
+* [Code in text](#code-in-text)
+  * [Code string example](#code-string-example)
+  * [Code block example](#code-block-example)
+* [Bullet points and numbered lists](#bullet-points-and-numbered-lists)
+* [Tables](#tables)
+
+[Use of Links](#use-of-links)
+* [Linking to other sites](#linking-to-other-sites)
+* [Linking to other documents](#linking-to-other-documents)
+
+[Use of Images](#use-of-images)
+
+
+### Introduction
+This Documentation styleguide describes what is documented in the Carbon repository and which format this  takes to enable you to contribute documentation in line with the current styles. It includes:
+- where documents live within the repository;
+- which format is used for which purpose;
+- language and grammar style;
+- linking to other documents and sites; and
```

Comment:
```suggestion
- linking to other documents and sites;
```

Reply:
I followed the first example in "A list of very short points" in this guide for bullet point lists: http://www.plainenglish.co.uk/punctuating-bulleted-lists.html. However, if you think the second example without an "and" before the last point, then I'm happy to change.

Reply:
Removed the "and" in all lists in this document.